collections/_article/godot-xr-update-feb-2025.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Godot XR update - February 2025"
+excerpt: "New plugin releases for Godot XR."
+categories: ["progress-report"]
+author: Bastiaan Olij
+image: /storage/blog/covers/february-2025-xr-progress-update.webp
+image_caption_description: "TBD"
+date: 2025-02-15 12:00:00
+---
+
+## Announcing the Meta Toolkit extension
+
+We are pleased to announce the release of the [Godot Meta Toolkit](https://github.com/godot-sdk-integrations/godot-meta-toolkit), a GDExtension plugin that exposes Meta's [Platform SDK](https://developers.meta.com/horizon/documentation/native/ps-platform-intro) and provides other tools to simplify and accelerate XR development on Meta's platform.
+
+The project is Open Source (MIT license) and maintained by [W4 Games](https://www.w4games.com/) with sponsorship from Meta. Contributions from the community are welcome! 🙂
+
+You can download version 1.0.1 [from GitHub](https://github.com/godot-sdk-integrations/godot-meta-toolkit/releases/tag/1.0.1-stable) or the [Asset Library](https://godotengine.org/asset-library/asset/3673).
+
+Let's take a look at the features available in this release!
+
+### Platform SDK
+
+The Platform SDK allows developers to access features of Meta's store and online services, including:
+
+- User profiles (including authentication and checking entitlement)
+- In-App Purchases (IAP)
+- Downloadable Content (DLC)
+- Friends, Parties, and Group Presence
+- Achievements
+- Leaderboards
+- ... and much more!
+
+Support of the Platform SDK in the Godot Meta Toolkit is done using code generation, which automatically generates the Godot classes by processing the [Platform SDK’s official C headers](https://developers.meta.com/horizon/downloads/package/oculus-platform-sdk/). This approach allows to easily keep the Godot Meta Toolkit up-to-date with the latest updates to the Platform SDK, and provides Godot APIs which match the Platform SDK's C, Unity and Unreal APIs.
+
+As of version 1.0.1, the Godot Meta Toolkit supports v71 of the Platform SDK.
```

Comment:
Does this line need to be changed too?

Reply:
Oh, yep, thanks for catching that! And that needs another change too:

```suggestion
As of version 1.0.2, the Godot Meta Toolkit supports v72 of the Platform SDK.
```

Part of the reason for this release is support of the latest Platform SDK version.